### DISCUSSION TITLE
Can i change default z-index?

### DISCUSSION THREAD

**Discussion by DmitryShcheklein** [fancyapps/ui#534] [Q&A]
I didn't see it in the documentation

I'm using it now:
```
.fancybox__container {
 z-index: 8000!important;
}
```

**Comment by fancyapps** [comment]
Well, yes, that is right.

**Reply by DmitryShcheklein** [reply]
@fancyapps, looks like a crutch.  Why don't use css variables like: `--fancybox-zIndex: 10000` ?

**Reply by fancyapps** [reply]
Thanks for the idea, I'll include that in the next release.
